src/app/helpers/isAmpPath.test.js

```diff
@@ -1,13 +1,10 @@
 import isAmpPath from './isAmpPath';
 
 describe('isAmpPath', () => {
-  it('should return false when passed a non-article path', () => {
-    expect(isAmpPath('/random')).toEqual(false);
```

Comment:
You have deleted the test example of a failure case. Having a failure case is important in a unit test file. We want to make sure that if we pass in an id that is not of the correct format, that the function returns `false`.